http.go

```diff
@@ -121,12 +128,25 @@ func (s *LumbermillServer) checkAuth(r *http.Request) error {
 	user := userPassParts[0]
 	pass := userPassParts[1]
 
-	if string(user) != User {
```

Comment:
Why not make line #123 `userPassParts := bytes.SplitN(decodedUserPass, []byte{':'}, 2)` be this instead `userPassParts := strings.SplitN(string(decodedUserPass), ':', 2)` instead of having to continually convert types down here?